cterasdk/core/users.py

```diff
@@ -12,7 +12,7 @@ class Users(BaseCommand):
     Portal User Management APIs
     """
 
-    _default_fields = ['name']
```

Comment:
Why did you remove the leading underscore? It's there to tell that it is a private class member

Reply:
I am fine with either. In core/devices we use **default** - I wanted to align to that. But i'm fine with the other way around.

Reply:
OK, leave it as you did